### PR TITLE
Build and github draft release

### PR DESCRIPTION
**Pull request by Ashersam**
Added changed in the package.json creates a win and Linux file simultaneously with .yml file(used for auto-update) and makes a draft release on the repo.
Tested for windows and Linux with a private repo
https://github.com/Ashersam/Test-Autographa/releases

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,110 +1,119 @@
...
-    "electron-builder": "^21.2.0",
-    "electron-packager": "^13.1.1",
-    "electron-rebuild": "^1.8.4",
-    "wait-on": "^3.2.0"
-  }
+	"name": "autographa-lite",
+	"description": "A Bible translation editor for everyone.",
+	"version": "1.3.4",
+	"main": "public/electron.js",
+	"author": "Friends of Agape <mathew.joel@protonmail.com>",
```
Comment (joelthe1) [inline package.json]: Can we change this email ID back to `mail@joelmath.io`?

File: package.json [hunk 2]
```diff
@@ -1,110 +1,119 @@
...
+		"@craco/craco": "^3.6.0",
+		"@material-ui/core": "^4.2.0",
+		"@material-ui/icons": "^4.2.1",
+		"chai-as-promised": "^7.1.1",
+		"customize-cra": "^0.2.12",
+		"diff-match-patch": "^1.0.0",
+		"electron-squirrel-startup": "^1.0.0",
+		"electron-log": "^2.2.7",
+		"fs-jetpack": "^1.0.0",
+		"jquery": "^3.3.1",
```
Comment (joelthe1) [inline package.json]: Maybe not in this PR but we should see if we can get rid of `jquery` and possibly other such obsolete dependencies.

File: package.json (hunk 1 above)
Comment (Ashersam) [reply package.json]: so for live and lite you are using different mail ID?

File: package.json (hunk 2 above)
Comment (Ashersam) [reply package.json]: yes I will check and clear those

Comment (Ashersam) [reply package.json]: jquery is a kind of peer dependency for bootstrap-slider. So we can remove until bootstrap slider exists

**Pull request merged**